### DISCUSSION TITLE
Win executable v1.0

### DISCUSSION THREAD

**Discussion by SweerItTer** [SweerItTer/BilibiliCartoonDownloader#2] [Announcements]
-----------------CN------------------
该可执行程序为基于 2024.3.10 完成的脚本(UI版)打包而成
带命令窗口(该阶段存在大量BUG,需要来查看报错)
基础功能完善,姑且可以用

后续可能会修复bug并推出更多平台的版本

使用方法:解压,运行main.exe
目前暂不支持保持登录,取消下载,自动检测下载历史续传

总而言之:存在各种问题,但是可以用

------------------EN------------------

Executable packaged for a UI script completed on March 10, 2024.
Includes a command window (due to numerous existing bugs requiring error monitoring).
Basic functionality is improved and somewhat usable.
Possible bug fixes and releases for more platforms in the future.

Usage: Unzip and run main.exe.
Currently does not support persistent login, canceling downloads, or automatic detection of download history for resuming.

In summary: There are various issues, but it's usable.

<hr /><em>This discussion was created from the release <a href='https://github.com/SweerItTer/BilibiliCartoonDownloader/releases/tag/'>Win executable v1.0</a>.</em>
